Cargo.toml

```diff
@@ -14,10 +16,12 @@ uuid = { version = "1.8.0", features = ["serde"] }
 serde = { version = "1.0", features = ["derive"] }
 thiserror = "1.0.58"
 rusqlite = { version = "0.29.0", features = [ "bundled", "blob" ] }
-tokio = { version = "1.37.0" }
+tokio = { version = "1.37.0", default-features = false }
```

Comment:
The `default-features = false` part has no effect as `tokio` doesn't declare default features, but I thought it's a good indicator for the desire to take full control of any feature that is used.